engine/impl/Playlist.php

```diff
@@ -538,16 +568,19 @@ public function updateTrack($playlistId, $id, $tag, $artist, $track, $album, $la
         return $success;
     }
 
-    public function insertTrackEntry($playlist, PlaylistEntry $entry, $wantTimestamp) {
+    public function insertTrackEntry($playlist, PlaylistEntry $entry, $timeStamp, &$status) {
         $id = 0;
         $success = $this->insertTrack($playlist,
                                       $entry->getTag(), $entry->getArtist(),
                                       $entry->getTrack(), $entry->getAlbum(),
                                       $entry->getLabel(),
-                                      $wantTimestamp,
-                                      $id);
+                                      $timeStamp,
```

Comment:
PlaylistEntry already contains a timestamp, so this new one in the function declaration is redundant.

Please replace $timeStamp with $entry->getCreated(), and remove the $timeStamp parameter from the function signature.

Reply:
this value is the spin timestamp and thus it is required. it will be renamed for clarification.

Reply:
Sure, but the point is that you can pass the spin timestamp inside PlaylistEntry.  It already contains a timestamp (get/setCreated)

Reply:
done.